server/api/user/user.controller.js

```diff
@@ -340,44 +336,43 @@ exports.authCallback = function(req, res, next) {
 exports.attendance = function(req,res){
     var userId = req.params.id;
     var userCode = req.body.code;
-    console.log("userCode",userCode);
-    console.log("process.env.RCOSDAYCODE",process.env.RCOSDAYCODE);
-    if (! process.env.RCOSDAYCODE){
-
-        res.send(500, "No day code yet!");
-
-    }
-    else if (!userCode || userCode.toUpperCase() != process.env.RCOSDAYCODE.toUpperCase()){
-
-        res.send(500, "Day Code Incorrect!");
-
-    }else{
-
-        var result = User.update({
-            _id: userId
-        },{
-            $push: {
-                attendance: new Date()
-            }
-        }, function(err){
-            if (err) return res.send(500, err);
-            res.send({"success":(err !== 0)});
-
-        });
-    }
+   
```

Comment:
This should verify that the daycode is still valid (same day, or time range of dates)